xbmc/platform/linux/powermanagement/GBMDPMSSupport.cpp

```diff
@@ -0,0 +1,61 @@
+/*
+ *      Copyright (C) 2009-2013 Team XBMC
+ *      http://kodi.tv
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with XBMC; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "GBMDPMSSupport.h"
+
+#include "ServiceBroker.h"
+#include "windowing/gbm/WinSystemGbm.h"
+
+void CGBMDPMSSupport::Register()
+{
+  CServiceBroker::RegisterDPMS(std::make_shared<CGBMDPMSSupport>());
```

Comment:
We should not transfer ownership to ServiceBroker. Some component has to own the object, not ServiceBroker.

Reply:
humm, do you have an idea who should own it? CApplication?